pkg/config/config.go

```diff
@@ -730,3 +731,30 @@ func (lci *LocalConfigInfo) GetOSSourcePath() (path string, err error) {
 
 	return sourceOSPath, err
 }
+
+// IsValidArgumentList validates the argument list for `odo config` command
+func IsValidArgumentList(args []string) error {
```

Comment:
This function should actually be merged with the `Args` function that is passed to the command struct when it's created since otherwise the check on args length will never occur (as it's done in that function). Actually, there would be merits to check the args there instead of in `Complete`…